agent/run.sh

```diff
@@ -29,6 +29,11 @@ export CATTLE_HOME=${CATTLE_HOME:-/var/lib/cattle}
 
 check_debug
 
+RUNNING_IMAGE="$(docker inspect $(hostname)| jq -r .[].Config.Image)"
```

Comment:
`rancher-agent` runs in `--net=host` so this won't work.  Registration and Upgrade run in `--net=bridge`.